functions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "functions",
-  "description": "Nexmo SMS on Firebase Cloud Functions",
+  "description": "Vonage SMS on Firebase Cloud Functions",
   "author": "Kelly J Andrews (kelly.andrews@vonage.com)",
```

Comment:
Make sure to add a contributors section and add your info.

Reply:
contributor added. Thanks for your review.